data/soapvalidator/MailClient/HAB/ZCS-5708-ModifyHABGroupRequest.xml

```diff
@@ -0,0 +1,626 @@
+			</t:response>
+		</t:test>
+		
+		<t:test>
+			<t:request>
+				<ModifyHABGroupRequest xmlns="urn:zimbraAdmin">
+					<habGroupOperation habGroupId="${hab.group8_id}" currentParentHabGroupId="${hab.group1_id}"	targetParentHabGroupId="${hab.group10_id}" op="move" />
+				</ModifyHABGroupRequest>		
+			</t:request>
+			<t:response>
+				<t:select path="//soap:Reason/soap:Text" match="invalid request: Target group:${hab.group10_name}@${domain.name} is a member of the group:${hab.group8_name}@${domain.name} being moved"/>
+			</t:response>
+		</t:test>	
+	</t:test_case>
+	
+	<t:test_case testcaseid="ModifyGroup5" type="deprecated"
+		bugids="ZCS-5708">
+		<t:objective>Move static API group from QA group to Ou2->Myorg
+		</t:objective>
+		<t:steps>
+			1. Trigger modifyHabrequest to move webUx group to myOrg group
+			2. Verify request should fail.
+		</t:steps>
+		
+		<t:test id="admin_login" required="true">
+			<t:request>
+				<AuthRequest xmlns="urn:zimbraAdmin">
+					<name>${admin.user}</name>
+					<password>${admin.password}</password>
+				</AuthRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:AuthResponse/admin:authToken" set="authToken" />
+			</t:response>
+		</t:test>
+		
+		<t:test>
+			<t:request>
+				<ModifyHABGroupRequest xmlns="urn:zimbraAdmin">
+					<habGroupOperation habGroupId="${hab.group4_id}" currentParentHabGroupId="${hab.group2_id}"	targetParentHabGroupId="${hab.group1AltOU_ou2_id}" op="move" />
+				</ModifyHABGroupRequest>		
+			</t:request>
+			<t:response>
+				<t:select path="//admin:ModifyHABGroupResponse/admin:parentHABGroup" attr="id" match="${hab.group1AltOU_ou2_id}"/>
+				<t:select path="//admin:ModifyHABGroupResponse/admin:parentHABGroup//admin:member" match="${hab.group4_name}@${domain.name}"/>
+			</t:response>
+		</t:test>	
+	</t:test_case>
+
+	<t:test_case testcaseid="ModifyGroup6" type="smoke"
+		bugids="ZCS-5708">
+		<t:objective>Move static Support group from MyOrg group to Dynamic API_dev group
+		</t:objective>
+		<t:steps>
+			1. Trigger modifyHabrequest to move Support group to API_dev group
+			2. Verify request should fail.
+		</t:steps>
+		
+		<t:test id="admin_login" required="true">
+			<t:request>
+				<AuthRequest xmlns="urn:zimbraAdmin">
+					<name>${admin.user}</name>
+					<password>${admin.password}</password>
+				</AuthRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:AuthResponse/admin:authToken" set="authToken" />
+			</t:response>
+		</t:test>
+		
+		<t:test>
+			<t:request>
+				<ModifyHABGroupRequest xmlns="urn:zimbraAdmin">
+					<habGroupOperation habGroupId="${hab.group10_id}" currentParentHabGroupId="${hab.group1_id}"	targetParentHabGroupId="${hab.group8_id}" op="move" />
+				</ModifyHABGroupRequest>		
+			</t:request>
+			<t:response>
+				<t:select path="//soap:Reason/soap:Text" match="invalid request: Target group cannot be dynamic."/>
+			</t:response>
+		</t:test>	
+	</t:test_case>
+
+</t:tests>	
+	
```

Comment:
Can we add a negative case where currentParentId is of some other group?

For ex:

<habGroupOperation habGroupId="${hab.group8_id}" currentParentHabGroupId="${hab.group3_id}"	targetParentHabGroupId="${hab.group6_id}" op="move" />